compiler-rt/lib/scudo/standalone/primary32.h

```diff
@@ -191,13 +191,15 @@ template <typename Config> class SizeClassAllocator32 {
     return BlockSize > PageSize;
   }
 
-  u16 popBlocks(CacheT *C, uptr ClassId, CompactPtrT *ToArray) {
+  u16 popBlocks(CacheT *C, uptr ClassId, CompactPtrT *ToArray,
+                const u16 MaxBlockCount) {
```

Comment:
It doesn't appear this value is used any place, is that expected?

Reply:
Yes, it'll be used in the case that TransferBatch  stores more blocks than `MaxBlockCount`. Now the TransferBatch almost always stores the same amount as `MaxBlocksCount` so we only use it in the DCHECK at this moment

Reply:
Do you need to add a [[maybe_unused]] in front of the variable? It seems like this could result in a build error when DCHECK is compiled out.

Reply:
You're right. Given that the same check has been done in `refill (local_cache.h)`, I just mark it as `UNUSED` and put a comment.